### PR TITLE
test/librbd: add get_group test

### PR DESCRIPTION
**Pull request by songtongshuai**
Signed-off-by: songtongshuai <songtongshuai@cmss.chinamobile.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by songtongshuai** [comment]
@trociny  Could you help review my code? please  T _ T

File: src/test/librbd/test_Groups.cc [hunk 1]
```diff
@@ -100,6 +100,10 @@ TEST_F(TestGroup, add_image)
   ASSERT_EQ(0, rbd_group_image_add(ioctx, group_name, ioctx,
                                    m_image_name.c_str()));
 
+  rbd_group_info_t group_info;
+  ASSERT_EQ(-ERANGE, rbd_get_group(image, &group_info, 0));
+  ASSERT_EQ(0, rbd_get_group(image, &group_info, sizeof(group_info)));
+
```
Comment (trociny) [inline src/test/librbd/test_Groups.cc:106]: nit: you could also test `rbd_get_group` before `rbd_group_image_add` to check that it returns an error as expected.

**Review by trociny — COMMENTED** [review]
Could you also add `image.get_group` test to `add_imagePP` in the same way?

And it would change the commit log message to: "test/librbd: add get_group test".

Otherwise LGTM. Thanks.

**Comment by trociny** [comment]
@songtongshuai Just FYI, you don't have to close an older PR just to open a new one with squashed commits. Just push your local branch with "force" option.

**Pull request closed without merging**